### ISSUE TITLE
Introducing myself

### ISSUE THREAD

**Issue by Ufitamahoro** [lab-brussels-1/home#141]
[ ] Create a repo and name it "Introducing-my-self"
[ ] Create a new issue and write a description of what I need to do
[ ] Clone the repo to my local machine
[ ] Create a new branch and name it 1-info
[ ] Check out the branch "1-info"
[ ] Add some info about me to "README.md" with my picture
[ ] Add my changes to the staging area
[ ] Commit my changes
[ ] Push my changes to gitHub
[ ] Merge my changes to the master or main branch
[ ] Link my issue to the pull request
[ ] Share my repo link with my colleagues and learn from each other

**Comment by danielhalasz** [comment]
# Week 1

## I Need Help With:

## What went well?
I get helped by Tamer and Samir, I got to learn what my mistakes are.

## What went less well?
It was confusing.

## Lessons Learned
- To reach out, network. 
- Linting
- Pull and push 

## Sunday Prep Work
- I am going to start

**Comment by danielhalasz** [comment]
@Ufitamahoro I have moved you checkin comment and my comment here and closing the other issue...please read below

@Ufitamahoro the check-in issue is intended to help keep track of your study progress within each module, and then each week you can add the corresponding label (now it is week-1) to the issue. So there is only one check-in issue per person per module. Within the check-in issue, you should list all the study material that you will go through and gradually tick off the boxes once you complete them, so we see what you have already completed and help accordingly.
Can you please explain what was confusing this week?

**Comment by Ufitamahoro** [comment]
Thank you @danielhalasz .
For this week, first I was confused when after cloning my repo, I was running the npm commands and get an error message. After some help, I got to understand that I didn't install node and npm. What I did then.

**Comment by danielhalasz** [comment]
@Ufitamahoro yes, it is important to install everything that we listed in the precourse module

**Comment by Ufitamahoro** [comment]
Week 2

## I Need Help With:
- Following up with branches. I know how to create them but once created I get confused with keeping up with them
- I don't understand the process of updating my Readme in the group repo

## What went well?
-Cloning the repo,  working on the cloned repo, save all the changes and pushing them 

## What went less well?
-Branching

## Lessons Learned
-Linting
-Working with VS code

## Sunday Prep Work
-I am learning more with Agile. It goes a bit quicker because I have worked with Agile so I know a little bit of it

**Comment by danielhalasz** [comment]
@Ufitamahoro 

- what do you mean by keeping up with the branches?
- what do you not understand? it is a simple .md file that you can edit in VSCode
- if you carefully check the repo, you will find a file called `cspell.json` and you can edit it to add new words to it
